.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Python CI
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+                
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      env:
+        PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
+      run: |
+        pip install -r requirements.txt
+        pip install -r requirements_dev.txt
```

Comment:
You could combine this into a single line. You may also want to add `pip install .` to install `wps-tools` itself.